certified-connectors/Fliplet/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API token",
+          "description": "The API token for this api",
+          "tooltip": "Provide your API token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#7e4c77",
+    "capabilities": [],
+    "publisher": "Tali Oussama"
```

Comment:
@squallstar for all our verified publishers we ask to have "publisher" and stackOwners properties the same. 
"publisher": "Name of the owner of the service or product/Name of company or organization.",
"stackOwner": "Name of the owner of the service or product/Name of company or organization".